### PR TITLE
Add default command

### LINKED ISSUE #8: Info CMD Not Working
This plugin is dope! But at least on Dokku 7.2, `acl:info` doesn't do anything.

### PR DESCRIPTION
**Pull request by scjody**
Add a default command that displays help. Fixes #8.

@bpostlethwaite Please review.

```
root@dokku:~# dokku acl
Usage: dokku acl[:COMMAND]

Commands:
  acl:add <app> <user>      Allow user to push to this repository
  acl:list <app>            Show list of users with access to repository
  acl:remove <app> <user>   Revoke users access to the repository
```

### CODE REVIEW

**Comment by bpostlethwaite** [comment]
:dancer:

File: functions [hunk 1]
```diff
@@ -0,0 +1,14 @@
+help_formatted() {
+  echo -e "Usage: dokku acl[:COMMAND]"
+  echo ''
+  echo 'Commands:'
+  help_content_func | sort | column -c2 -t -s,
```
Comment (josegonzalez) [inline functions]: This isn't the exact format. Example incoming.

Comment (josegonzalez) [reply functions]: https://github.com/dokku/dokku/blob/master/plugins/nginx-vhosts/internal-functions

Comment (scjody) [reply functions]: OK now?

**Comment by josegonzalez** [comment]
What does the output look like when you do:

```
dokku help
dokku acl
dokku acl:help
```

Does it show up correctly?

**Comment by scjody** [comment]
@josegonzalez `dokku acl` and `dokku acl:help` are identical and look good to me. Please let me know if you want any changes:

```
root@dokku:~# dokku acl
Usage: dokku acl[:COMMAND]

Manage app-level Access Control Lists.

Additional commands:
  acl:add <app> <user>      Allow user to push to this repository
  acl:list <app>            Show list of users with access to repository
  acl:remove <app> <user>   Revoke users access to the repository
root@dokku:~# dokku acl:help
Usage: dokku acl[:COMMAND]

Manage app-level Access Control Lists.

Additional commands:
  acl:add <app> <user>      Allow user to push to this repository
  acl:list <app>            Show list of users with access to repository
  acl:remove <app> <user>   Revoke users access to the repository
```

`dokku help` gives:
```
root@dokku:~# dokku help
Usage: dokku [--quiet|--trace|--rm-container|--rm|--force] COMMAND <app> [command-specific-options]

Primary help options, type "dokku COMMAND:help" for more details, or dokku help --all to see all commands.

Commands:

    apps       Manage Dokku apps
    certs      Manage Dokku apps SSL (TLS) certs
    checks     Manage zero-downtime settings
    config     Display all global or app-specific config vars
    domains    Manage vhost domains used by the Dokku proxy
    enter      Connect to a specific app container
    events     Show the last events (-t follows)
    help       Print the list of commands
    logs       Show the last logs for an application
    ls         Pretty listing of deployed applications and containers
    nginx      Interact with Dokku's Nginx proxy
    proxy      Manage the proxy used by dokku on a per app
    ps         List processes running in app container(s)
    run        Run a command in the environment of an application
    shell      Spawn dokku shell
    ssh-keys   Manage public ssh keys that are allowed to connect to Dokku
    storage    Mount local volume / directories inside containers
    tags       List all app image tags
    tar        Deploy applications via tarball instead of git
    trace      Enable dokku tracing
    url        Show the first URL for an application (compatibility)
    urls       Show all URLs for an application
    version    Print dokku's version

Community plugin commands:

    acl   Plugin for adding the ability to restrict push privileges for app to certain users
```

Should the description for the `acl` command be aligned with the other descriptions? Should it end at column 80 for users with narrower terminals? That message in particular is unchanged in this PR, but I'm happy to bring it up to modern standards as long as I know what they are 😸.

**Comment by josegonzalez** [comment]
This is fine. What about `dokku help --all`?

**Comment by scjody** [comment]
The relevant part of `dokku help --all`:

```
Community plugin commands:

  acl:add <app> <user>      Allow user to push to this repository
  acl:list <app>            Show list of users with access to repository
  acl:remove <app> <user>   Revoke users access to the repository
```

Now that I look at it again, the grammar and wording could be a bit better so I'll fix that quickly.

**Comment by scjody** [comment]
Latest versions:

`dokku acl:help` and `dokku acl`:

```
Usage: dokku acl[:COMMAND]

Manage app-level Access Control Lists.

Additional commands:
  acl:add <app> <user>      Allow <user> to push to <app>'s repository
  acl:list <app>            Show list of users with access to <app>'s repository
  acl:remove <app> <user>   Revoke <user>'s access to <app>'s repository
```

Our part of `dokku help`:

```
Community plugin commands:

    acl   Plugin to manage app-level Access Control Lists
```

Our part of `dokku help --all`:

```
Community plugin commands:

  acl:add <app> <user>      Allow <user> to push to <app>'s repository
  acl:list <app>            Show list of users with access to <app>'s repository
  acl:remove <app> <user>   Revoke <user>'s access to <app>'s repository
```

**Review by josegonzalez — APPROVED** [review]

**Pull request merged**